core/src/main/scala/kafka/coordinator/GroupMetadataManager.scala

```diff
@@ -141,7 +141,8 @@ class GroupMetadataManager(val brokerId: Int,
       // if we crash or leaders move) since the new leaders will still expire the consumers with heartbeat and
       // retry removing this group.
       val groupPartition = partitionFor(group.groupId)
-      val tombstone = new Message(bytes = null, key = GroupMetadataManager.groupMetadataKey(group.groupId))
+      val tombstone = new Message(bytes = null, key = GroupMetadataManager.groupMetadataKey(group.groupId),
+        timestamp = SystemTime.milliseconds, magicValue = Message.MagicValue_V1)
```

Comment:
Hi Jun, we are doing message format conversion, right? If another broker is on old code and sends FetchRequest v1, we will down convert the message to v0. So it should not break even if leader has message format v1 on disk and follower is fetching using inter-broker protocol version before 0.10.0.

I am wondering if we should simply let the internal topic message format comply with inter-broker protocol version. The reason is this guarantees no message format conversion is needed for internal topic replication. And it avoids manually setting message format version config for the internal topic.